bxserum/provider/http.go

```diff
@@ -111,6 +111,17 @@ func (h *HTTPClient) GetUnsettled(market string, owner string) (*pb.GetUnsettled
 	return result, nil
 }
 
+// GetAccountBalance returns all OpenOrders accounts for a given market with the amounts of unsettled funds
+func (h *HTTPClient) GetAccountBalance(owner string) (*pb.GetAccountBalanceResponse, error) {
+	url := fmt.Sprintf("%s/api/v1/account/balance?ownerAddress=%s", h.baseURL, owner)
+	result := new(pb.GetAccountBalanceResponse)
```

Comment:
I'm actually a little surprised this works in our codebase. Something like this doesn't work for `json.Unmarshal`, since this is a pointer pointing to no allocated data, so there's no data structure to marshal the data to, but I suppose `protojson` must do something special to get this working.